.ci.yaml

```diff
@@ -60,6 +62,88 @@ targets:
         ]
     scheduler: luci
 
+  - name: windows-build+platform-tests master
```

Comment:
target names must follow the format $platform $name

So `Windows build+platform-tests master`. I'll send a PR to update this in the CI_YAML.md

Reply:
Fixed.